File: 백준/Bronze/2525. 오븐 시계/오븐 시계.c
int main()
{
    int a, b; // a시 b분
    int c; // time for cooking
    
    scanf("%d %d\n", &a, &b);
    scanf("%d", &c);
    
   if((b + c) < 60) {
        printf("%d %d", a, b+c);
   }
   else{
        int h = (b + c) / 60; // 더해질 시간
        int m = (b + c) % 60;
        if((h + a) > 23){
            printf("%d %d", (h+a-24), m);
        } 
        else{
            printf("%d %d", h+a, m);
        }
   }
    
    return 0;
}

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let [hour, minute] = input[0].split(' ').map(Number);;
let time = Number(input[1]);

minute += time; 
hour += parseInt(minute / 60);
minute = minute % 60;

if (hour >= 24) {
  hour = hour % 24;
}

console.log(hour, minute);